pkg/accountdb/dbo/user.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbo
+
+import (
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"storj.io/storj/pkg/accountdb/dbx"
+	"time"
+)
+
+// Database object that describes User entity
+type User struct {
+	id uuid.UUID
+
+	firstName string
+	lastName string
+	email string
+	password string
```

Comment:
This should be `PasswordHash []byte`.

Reply:
I've found mixing authentication and user information bad idea. e.g. supporting multiple ways of logging in (Google, Facebook etc.)... However, it's a good solution for now, until the need arises.